controller/src/beerocks/master/tasks/client_steering_task.cpp

```diff
@@ -297,7 +297,4 @@ void client_steering_task::handle_task_end()
         database.update_node_11v_responsiveness(sta_mac, false);
     }
     database.set_node_handoff_flag(sta_mac, false);
-    if (!steer_restricted) {
-        son_actions::unblock_sta(database, cmdu_tx, sta_mac);
```

Comment:
Why did you remove it? please explain.

Reply:
In the FINALIZE state, the controller sent an unblock message to all agents, but it's not really needed due to the validity period, so I removed that part. @morantr

Reply:
> In the FINALIZE state, the controller sent an unblock message to all agents, but it's not really needed due to the validity period, so I removed that part. @morantr

The current bwl implementations don't use the validity period option, so that will have to be implemented first.

I think it's best if you un-remove this code for now, and create a new task to actually use the validity period in bwl. I shouldn't have included that bit in #432 - my bad.

Reply:
Ok 👍 @arnout

Reply:
#445